packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -135,6 +135,7 @@ export function getMetadataFormatVersion(metadata?: IContainerRuntimeMetadata):
     return metadata?.summaryFormatVersion ?? 0;
 }
 
+export const aliasBlobName = ".aliases";
```

Comment:
I'm not sure I see the value of these exports being here. It's details of particular implementation of IContainerRuntime, maybe some other implementation will use same summary logic, but not have these blobs (or blobs with different names)? Given that aliasBlobName is used only in containerRuntime.ts (and I'd guess others as well), I'd rather move them there.

Reply:
agree

Reply:
This should be done in a separate change though as some are used in other places refactored out of container runtime (like garbage collection) and other are exported indirectly to (again) places which have been refactored out of the container runtime (like nonDataStorePaths to datastores).